libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Static.xcconfig

```diff
@@ -11,4 +11,4 @@
 INSTALL_PATH = /
 SKIP_INSTALL = YES
 
-HEADER_SEARCH_PATHS = $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)
+// HEADER_SEARCH_PATHS = $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)
```

Comment:
I thought I'd removed it entirely, I'll add a commit for that.

Basically, the problem is that by adding BUILT_PRODUCTS_DIR/Headers directly to the import path, Xcode finds multiple headers defining the same symbol when building dependencies.  So the moment it encounters an #import of the umbrella header, it finds multiple alternatives and fails to compile.

Everything still compiles successfully without this line added.